defaults/main.yml

```diff
@@ -327,6 +330,16 @@ ember_docroot: "{{ ember_basedir }}/htdocs"
 # Set this to the user which apache runs as
 ember_apache_user: www-data
 
+# Keycloak URL
+#TODO trailing slash really needed?
+ember_oidc_host: "{{ camac_auth_idp_url }}/"
```

Comment:
@anehx do you know if the trailing slash is needed?

Reply:
No idea..

Reply:
Let's leave it as is then. I removed the TODO and added a note highlighting that `camac_auth_idp_url` should not have a trailing slash.